src/components/Registration/RegistrationForm.tsx

```diff
@@ -1,19 +1,27 @@
 import React from 'react';
-import { Formik, Form, Field, ErrorMessage, FormikHelpers } from 'formik';
+import { Formik, Form, ErrorMessage, FormikHelpers } from 'formik';
+import { useTranslation } from "react-i18next";
 
 import IRegistrationForm from './RegistrationForm.interface';
+import { SignupSchema } from './validate';
+import { Button, Label, Input, StyledInlineErrorMessageReg, IconPassword, IconText, FormGroup, Line, Foot } from "components/styles";
+import { LogoPageSmall } from "components/styles/LogoPage.style";
 
 
-const registrationValidation = (values:any) => {
-  const errors = {};
-  return errors;
-}
-
 export const RegistrationForm = () => {
+  const { t } = useTranslation();
+
   return (
     <Formik
-    initialValues={{ userName: '', firstName: '', lastName: '', email: '', password: '', confirmPassword: '' }}
-    validate={registrationValidation}
+    initialValues={{ 
+      userName: '', 
+      firstName: '', 
+      lastName: '', 
+      email: '', 
+      password: '', 
+      confirmPassword: '' 
+    }}
+    validationSchema={SignupSchema}
```

Comment:
w takim razie musiała by to być funkcja która przyjmuje parametr t
SignupSchema(t)